python/pyarrow/array.pxi

```diff
@@ -599,6 +599,20 @@ cdef class Array:
                                               self, &out))
         return wrap_array_output(out)
 
+    def to_numpy(self):
+        """
+        Convert to a NumPy array
```

Comment:
The docstring should mention the zero-copy nature. For example "Construct a Numpy view of this array".

Reply:
Done